chef-powershell/Gemfile.lock

```diff
@@ -1,35 +1,35 @@
 PATH
```

Comment:
Is there a reason to include the gemfile.lock in this repo? cc @lamont-granquist

Reply:
Oh truth. I'll remove it.